config/aliases.js

```diff
@@ -29,7 +29,7 @@ function findAliases(plugins) {
  */
 function changePaths(hashmap) {
 	Object.keys(hashmap).forEach(key => {
-		hashmap[key] = path.join(__dirname, '../packages', hashmap[key].replace('./packages', ''));
+		hashmap[key] = path.join(__dirname, '../', hashmap[key].replace('./packages', ''));
```

Comment:
This was done to fix the following error message: 
`Error: Cannot find module '/Users/Alex/Projects/inferno/**packages**/src/testing/inferno-component'`,
The correct path was:
`'/Users/Alex/Projects/inferno/src/testing/inferno-component'`
After changing this build still compiles and tests still pass but I'd like to be sure that there are no hidden consequences of this change.